src/app/view/components/_common/searchable-dropdown/component/searchable-dropdown.component.spec.ts

```diff
@@ -21,7 +21,7 @@ describe('SearchableDropdownComponent', () => {
     let pageLinkSize: number;
     let mainButton: DebugElement;
 
-    beforeEach(async(() => {
+    beforeEach(() => {
```

Comment:
Why you remove the async? we need it because the component have external template.